osmosis/README.md

```diff
@@ -145,14 +145,14 @@ const [stateValue, setStateValue, isHydrated] = usePersistedState(initialValue,
 
 Where the hook params are:
 
-- **initialValue**: the initial value to use for this state, just like from `useState`. This only initializes the state value at run time. If present, the initial value will be overridden by any persisted state that is rehydrated on mount.
+- **initialValue**: the initial value to use for this state, just like from `useState`. This only initializes the state value at run time. If present, the initial value will be overridden by any persisted state that is rehydrated on the mount.
 
 - **persistenceKey**: the key to be passed to the configured `setItem` function to store the value in the persistence layer.
 
 And the return params are:
 
 - **stateValue**: the value as stored in state, just like from `useState`.
 
-- **setStateValue**: the function to update the value in state. This is almost identical to the function returned from `useState`. The only difference is that in addition to setting the current value in state, it also asynchronously calls the configured `setItem` function to allow the user to store the latest state value in the persistence layer desired, using the `persistenceKey` supplied.
+- **setStateValue**: the function to update the value in state. This is almost identical to the function returned from `useState`. The only difference is that in addition to setting the current value in the state, it also asynchronously calls the configured `setItem` function to allow the user to store the latest state value in the persistence layer desired, using the `persistenceKey` supplied.
```

Comment:
This one feels weird as well....I usually find myself talking about "state" and never describe it as "the state" like done here